ryr-docs/mkdocs.yml

```diff
@@ -9,6 +9,9 @@ pages:
   - Guides:
     - Setting up a local environment: 'guides/setup-full-environment.md'
     - Keeping your branch up-to-date: 'guides/update-branch.md'
+    - Git extras Installation: 'guides/git-extras.md'
+    - Hub Installation: 'guides/install-hub.md'
+    - Liquidprompt Installation: 'guides/install-liquidprompt.md'
```

Comment:
The titles don't match the document titles. They should also be `Installing Liquidprompt`, etc.